setup.py

```diff
@@ -25,7 +25,7 @@
 
 setup(
     name="metadataproxy",
-    version="1.3.1",
+    version="1.3.3",
```

Comment:
Sure thing, I wasn't even sure if you'd want me to touch the version in my PR. I mostly just bumped it so that my private index would pick up the new build in preference to pypi.